code/datums/helper_datums/teleport.dm

```diff
@@ -170,25 +176,41 @@
 							var/obj/item/organ/external/E = pick(organs_to_gib)
 							to_chat(H, "<span class='danger'>You partially phase into \the [impediment], causing your [E.name] to violently dematerialize!</span>")
 							E.droplimb(0,DROPLIMB_BLUNT)

```

Comment:
Teleporting into full tiles no longer causes you to explode, so why should partial teleportations cause it?

Reply:
They don’t. They just cause you to lose a limb.

Reply:
The limb... basically explodes.